keras_nlp/models/deberta_v3/deberta_v3_classifier_test.py

```diff
@@ -113,12 +113,27 @@ def test_classifier_fit_no_xla(self):
         self.classifier.fit(self.preprocessed_dataset)
 
     def test_serialization(self):
```

Comment:
Could the serialization and saving tests be combined? The later is a superset of the former and they are both pretty slow.

Reply:
Saving is still 2-3x slower than serialization, so the net effect of pushing more test cases into the saving path would actually be an even slower test suit.

Is you goal to make these faster or less code?

- If the former, we could consider other follow ups, like not making a dataset in `setUp`. That's an easy win that would actually speed things up.
- If the latter, let's just focus on consolidating our testing code! Something I would really prioritize, just lacking bandwidth.